### PR TITLE
wrong directory

### PR DESCRIPTION
**Pull request by kentcalero**
following the tutorial, it seems the folder should be /srv/homeassistant, not /svr/hass

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/getting-started/z-wave.markdown [hunk 1]
```diff
@@ -58,7 +58,7 @@ With this installation, your `config_path` needed below will resemble:
 
 If you followed along with setting up a virtual environment, your path will be:
 ```bash
-/srv/hass/python-openzwave/openzwave/config
+/srv/homeassistant/python-openzwave/openzwave/config
```
Comment (mnoorenberghe) [inline source/getting-started/z-wave.markdown]: This used to be correct if you followed the virtualenv guide but it was changed to this in 34d80da743a634fbc3175433f6cb2f49dc728f1e

Comment (balloob) [inline source/getting-started/z-wave.markdown]: That's good to note. We should standardize on `homeassistant`

**Comment by balloob** [comment]
Thanks!

**Comment by kentcalero** [comment]
I'm going to rerun the entire installation again tonight and see if anything else pops up. Will update anything I find.

Kent L. Calero

Sent from the field through my iPhone.

On Feb 18, 2017, at 9:03 PM, Paulus Schoutsen <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks!

**Comment by kentcalero** [comment]
are any of you available if I run into issues or if i have questions about the install procedures?

**Comment by mnoorenberghe** [comment]
The support channels are listed at https://home-assistant.io/help/ and I would recommend them over Github.

**Pull request merged**